code/modules/mob/mob_movement.dm

```diff
@@ -159,7 +160,7 @@
 	//backpedal and strafe slowdown for quick intent
 	if(L.fixedeye || L.tempfixeye)
 		if(L.dir != target_dir)
-			add_delay += 2
+			add_delay *= SQRT_2
```

Comment:
```suggestion
			add_delay *= sqrt(2)
```
see above about const-folding

Reply:
also this change makes no sense, this is nothing to do with diagonal moves, this is for backpedaling. you're just making backpedaling/strafing faster for basically no reason?